### DISCUSSION TITLE
모임 후기

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/9th-builder#3] [Announcements]
모임 후기 작성을 위한 공간입니다!

**Comment by seriousran** [comment]
온보딩 1주차 모임을 진행했습니다!
가짜연구소의 철학과 문화에 대해서 소개하는 시간을 가졌는데요! 😍 

WHAT에서 부터 시작하는 사고에서, WHY 중심적 사고로 바꾸는! 중요한 이야기와 함께, `우리가 하는 모든 것들은 성장이 멈추는 불편한 상황을 이겨내고, 남들과는 다르게, 더 깊이, 더 멀리 나아가는 것입니다.`라는 비전을 공유했습니다.

이번 9기 활동, 그저 형식적인 기수가 되는 것이 아니라- 정말 우리의 인생을 바꿀! 큰 성장을 이루는 하반기 보냅시다! 화이팅! 🔥 

Feat. [사이몬 시넥 - 왜 해야하는가?](https://youtu.be/mP5SYrK4fMQ?si=MgBz0TDyK3yTSeLo) 명강의
*댓글로 출석 확인합니다!

**Reply by suhyeong10** [reply]
이번 1주차 모임을 통해서 저는 어떤 사고 방식을 가지고 인공지능 공부를 시작했고 또 지금은 어떤 생각을 가지고 있는지 깊게 생각해볼 수 있었습니다 좋은 이야기 나누어 주셔서 감사합니다!

**Reply by WongiPark0628** [reply]
1주차 회의를 통해서, 스스로를 고찰해보는 계기가 되었습니다. 좋은 말씀해주셔서 진심으로 감사드립니다.

**Reply by kyeongchan92** [reply]
스터디가 한 기수 끝날 때마다 목표가 무엇이었는지, 다음 목표는 어떻게 설정해야 할 지에 대해 흐릿해질 때도 있었는데요, WHY로부터 시작되는 사고가 도움이 될 것 같다고 생각했습니다! 좋은 내용 공유해주셔서 정말 감사드리고, 스터디 가서도 그대로 전달하여 더욱 뚜렷한 목표를 세워보겠습니다~! 👍😄

**Reply by changh95** [reply]
가짜연구소의 철학과 비전에 대해 잘 이해할 수 있는 세션이였습니다. 감사합니다!

**Reply by thrcle** [reply]
성장이 불안감해소를 위한 수동적인 소비의 형태로 전락한다는 문구가 인상깊었습니다. 저는 ‘일단 뭐라도 해보자’는 마인드는 좋다고 생각하고 있었는데, 거기서 한단계 나아가서 계속 정신없고 바쁜데 왜하는지 모른다면 그것은 내가 불안하기 때문인거구나를 새삼 느꼈습니다.

또 당신이 무엇을 가져서 함께 하는 사람들이 아니라, 당신이 믿는 것을 믿어주는 사람들과 함께 하라는 문구도 인상깊었습니다. 그렇다면 나는 무엇을 믿고있지? 라는 자문을 던져보았을 때, 솔직히 당장 뾰족한 생각이 안들더라구요. 그만큼 왜에 대해 생각해본적이 별로 없었구나 깨닫게 되었습니다.

 회사 내에서 느꼈던 여러가지 불편함에 대해 익숙해지고, 무의식적으로 나의 한계를 짓고 싶지않아서 가짜연구소를 찾은 것이었는데,  신청하길 잘했다는 생각이 들었습니다. 난 대체 ‘뭐가 불편했는지’에 대해 정리해보면 저의 왜가 나오지 않을까 생각이 드네요. 앞으로의 활동도 잘 부탁드립니다!

**Reply by shstat1729** [reply]
박상호 - 이번 1주차 모임을 하면서, 제가 왜 가짜연구소 활동을 하는지, 왜 학위 과정을 하는지 돌아보는 시간이 되었습니다. 감사합니다.

**Reply by HangryDev** [reply]
정한결: 느껴지는게 많았어요. 마침 리더십에 대해 고찰하던 시즌이었거든요. 찬란님의 내용 공유 덕에 inspiring 하는 부분을 놓치고 있었다는걸 좀 깨닫게 되었죠. '영감을 주고 비전으로 초대하는 사람'이라고 생각하면서 활동해야겠습니다.

**Reply by potea** [reply]
지난 8기와 다르게 좀 더 체계적이고, 명확한 목표를 설정한 것 같습니다.
